MAUI/Button/Customization.md

```diff
@@ -0,0 +1,671 @@
+---
+layout: post
+title: Customization in .NET MAUI Button control | Syncfusion
+description: Learn here all about Customization support in Syncfusion .NET MAUI Button (SfButton) control and more.
+platform: maui
+control: Sfbutton
+documentation: ug
+---
+
+# Customization in .NET MAUI Button (SfButton)
+
+The `.NET MAUI Button` control supports to customize the border color, image width, corner radius, background color, and more. The button control can be customized using the following properties:
+
+## Text Customization
+
+The text inside the button can be customized by its text color, font size, font attributes, font family and text alignment.
+
+### TextColor
+
+The `TextColor` property is used to customize the color of text in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" TextColor = "White">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.TextColor = Color.White;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with text color](images/Button_textcolor.png)
+
+### FontSize
+
+The `FontSize` property is used to customize the size of text in `SfButton`.
```

Comment:
The `FontSize` property is used to customize the size of the text in `SfButton`.